### PR TITLE
Make

### PR DESCRIPTION
**Pull request by dougsland**
[Makefile: add video as req to window_manager](https://github.com/containers/qm/commit/3f3659ea6497d448d230ffdf6dcc6b9c13826d67)
[Makefile: reduce paylog in dist target](https://github.com/containers/qm/commit/fe4606a5184c1ee6d20d93887955241e983833e6)

### CODE REVIEW

**Comment by pengshanyu** [comment]
Looks good to me.

**Review by aesteve-rh — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -78,7 +85,7 @@ rpm: clean dist ##             - Creates a local RPM package, useful for develop
 ostree: qm_dropin_img_tempdir ##             - A helper for creating QM packages for ostree based distros
 
 .PHONY: qm_dropin_window_manager
-qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input ##         - QM RPM sub-package qm_dropin_window_manager
+qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input qm_dropin_mount_bind_video0 ##         - QM RPM sub-package qm_dropin_window_manager
 	sed -i 's/%define enable_qm_window_manager 0/%define enable_qm_window_manager 1/' ${SPECFILE}
```
Comment (nsednev) [inline Makefile:89]: I think that you may add an @ symbol before the sed and $(MAKE) commands to suppress the command output and only show the command if it fails.
`@sed -i 's/%define enable_qm_window_manager 0/%define enable_qm_window_manager 1/' ${SPECFILE}`

File: Makefile [hunk 2]
```diff
@@ -78,7 +85,7 @@ rpm: clean dist ##             - Creates a local RPM package, useful for develop
 ostree: qm_dropin_img_tempdir ##             - A helper for creating QM packages for ostree based distros
```
Comment (nsednev) [inline Makefile:85]: I think that the formatting of the comments may be refined to make them more consistent and easier to read.
Adding a ## symbol before the comments to make them more visible and distinguishable from the code:
`ostree: qm_dropin_img_tempdir
	## Helper for creating QM packages for ostree based distros`

File: Makefile [hunk 3]
```diff
@@ -78,7 +85,7 @@ rpm: clean dist ##             - Creates a local RPM package, useful for develop
...
 
 .PHONY: qm_dropin_window_manager
-qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input ##         - QM RPM sub-package qm_dropin_window_manager
+qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input qm_dropin_mount_bind_video0 ##         - QM RPM sub-package qm_dropin_window_manager
```
Comment (nsednev) [inline Makefile:88]: Same here:
`qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input qm_dropin_mount_bind_video0
	## QM RPM sub-package qm_dropin_window_manager`

File: Makefile (hunk 1 above)
Comment (Yarboa) [reply Makefile:89]: I proposed different solution 
in here https://github.com/containers/qm/pull/612
better to use macros
Packit, doe not use the makefile to build rpms

**Review by Yarboa — APPROVED** [review]
LGTM
I will rework it on here 
https://github.com/containers/qm/pull/612

**Comment by dougsland** [comment]
I will send a new one just to reduce the payload of make, there is a new solution regarding building the packages for packet.

**Pull request closed without merging**